src/AzurePipelinesToGitHubActionsConverter.Core/Conversion/StepsProcessing.cs

```diff
@@ -1012,67 +1031,62 @@ private GitHubActions.Step CreateFunctionalTestingStep(AzurePipelines.Step step)
             run += "Write-Host \"$command\"\n";
             run += "Invoke-Expression $command";
 
-            //To PowerShell script
+            // To PowerShell script
             step.script = run;
-            GitHubActions.Step gitHubStep = CreateScriptStep("powershell", step);
 
-            return gitHubStep;
+            return CreateScriptStep("powershell", step);
         }
 
         public GitHubActions.Step CreateSetupJavaStep(string javaVersion)
         {
-            GitHubActions.Step gitHubStep = new GitHubActions.Step
+            return new GitHubActions.Step
             {
                 name = "Setup JDK " + javaVersion,
                 uses = "actions/setup-java@v1",
-                with = new Dictionary<string, string>
+                with = new OrderedDictionary
                 {
                     { "java-version", javaVersion}
                 }
             };
-
-            return gitHubStep;
         }
 
         public GitHubActions.Step CreateSetupGradleStep()
         {
-            //Going to: 
-            //- name: Grant execute permission for gradlew
-            //  run: chmod +x gradlew
+            // Going to: 
+            // - name: Grant execute permission for gradlew
+            //   run: chmod +x gradlew
 
-            AzurePipelines.Step step = new Step
+            var step = new Step
             {
                 name = "Grant execute permission for gradlew",
                 script = "chmod +x gradlew"
             };
-            GitHubActions.Step gitHubStep = CreateScriptStep("", step);
 
-            return gitHubStep;
+            return CreateScriptStep("", step);
         }
 
         public GitHubActions.Step CreateGradleStep(AzurePipelines.Step step)
         {
-            //coming from:
-            //- task: Gradle@2
-            // inputs:
-            //   workingDirectory: ''
-            //   gradleWrapperFile: 'gradlew'
-            //   gradleOptions: '-Xmx3072m'
-            //   publishJUnitResults: false
-            //   testResultsFiles: '**/TEST-*.xml'
-            //   tasks: 'assembleDebug'
+            // coming from:
+            // - task: Gradle@2
+            //  inputs:
+            //    workingDirectory: ''
+            //    gradleWrapperFile: 'gradlew'
+            //    gradleOptions: '-Xmx3072m'
+            //    publishJUnitResults: false
+            //    testResultsFiles: '**/TEST-*.xml'
+            //    tasks: 'assembleDebug'
 
-            //Going to:
-            //- name: Build with Gradle
-            //  run: ./gradlew build
+            // Going to:
+            // - name: Build with Gradle
+            //   run: ./gradlew build
 
             step.script = "./gradlew build";
-            GitHubActions.Step gitHubStep = CreateScriptStep("", step);
 
-            return gitHubStep;
+            return CreateScriptStep("", step);
```

Comment:
Is this supposed to be an empty string?

Reply:
I guess it is, and I haven't changed the method call - just combined the return with the call rather than storing response in a local var and returning.